src/Tizen.Network.Connection/Tizen.Network.Connection/ConnectionProfile.cs

```diff
@@ -36,6 +37,8 @@ public class ConnectionProfile : IDisposable
 
         private Interop.ConnectionProfile.ProfileStateChangedCallback _profileChangedCallback;
 
+        private static TizenSynchronizationContext context = new TizenSynchronizationContext();
+
```

Comment:
IDisposable interface of ConnectionProfile was wrong implemented,

unmanaged resource (in this case ProfileHandle) should always released in Dispose(bool) method. 
managed resource is only released when disposing was true